tests/data/outputs/events.txt

```diff
@@ -0,0 +1 @@
+recv,pwrite64,pread64,sendfile64,read,write,writev,pwritev,readv,sendfile,preadv,sendto,recvfrom,send,sendmsg,recvmsg,sendmmsg,recvmmsg
```

Comment:
```suggestion
sendfile,pwritev,preadv,writev,read,pwrite,write,sendto,recv,send,sendmsg,recvfrom,recvmsg,pread,sendmmsg,recvmmsg,readv
```
The latest Falco dev fixed a couple of things, this should be the most up to date and hopefully definitive list.